tests/symlink/test_resolve_links.py

```diff
@@ -33,27 +33,28 @@ def create_symlinks(temp_dir):
 
 
 def test_symlinks():
```

Comment:
Does the pytest skipif decorator work? It might require importing `pytest`.

```suggestion
@pytest.mark.skipif(os.name != 'posix',
                    reason="requires posix os")
def test_symlinks():
```